Content.Shared/GridPreloader/Prototypes/PreloadedGridPrototype.cs

```diff
@@ -0,0 +1,15 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.GridPreloader.Prototypes;
+
+[Prototype("preloadedGrid")]
+public sealed partial class PreloadedGridPrototype : IPrototype
+{
+    [IdDataField] public string ID { get; } = string.Empty;
+
+    [DataField(required: true)]
+    public string? Path;
+
+    [DataField]
+    public int Copies = 1;
+}
```

Comment:
Needs documenting also pretty sure prototype strings are obsolete.

Reply:
also respath for `Path` instead of string

Reply:
think IdDataField still has to be a string for some reason along with le {get;}

Reply:
every prototypes use string
https://github.com/search?q=repo%3Aspace-wizards%2Fspace-station-14%20%20%5BIdDataField%5D%20&type=code